src/modules/welcome/Config.cpp

```diff
@@ -231,6 +235,7 @@ QString Config::donateUrl() const
 void Config::setDonateUrl(const QString& url)
 {
     m_donateUrl = url;
+    emit donateUrlChanged();
```

Comment:
These emits are fine, but **generally** these parts of the config should be read-only -- they are set once from the config file, and are not user-modifiable.

Reply:
but since the config object sets those properties too late for qml the signals are needed for qml to update the labels. if those properties were set right at the construction of the conf then it would be fine to remove them, no otherwise.